services/API-service/src/api/disaster-type/disaster-type.service.ts

```diff
@@ -14,6 +15,25 @@ export class DisasterTypeService {
   @InjectRepository(DisasterTypeEntity)
   private readonly disasterTypeRepository: Repository<DisasterTypeEntity>;
 
+  public async getDisasterType(
```

Comment:
nice. There's a lingering separate getDisasterType in admin-area.service, which can still point here as well.

Reply:
processed in commit